### DISCUSSION TITLE
PLEASE let us disable ALL keyboard shortcuts

### DISCUSSION THREAD

**Discussion by foresto** [community/community#113472] [Accessibility]
### Select Topic Area

Product Feedback

### Body

Dear GitHub,

Your keyboard shortcuts have been infuriating thorns in my side since they were first imposed, and they remain so even today.

The worst have been those that hijack my browser's built-in shortcuts, such as:

- Blocking Firefox's built-in Control+G shortcut for find-next (reported as #61100).
- Making PageDown jump to the bottom of a 4000 line file instead of scrolling down one page.
- Making Home and End move a text cursor (which I don't want in the first place) instead of scrolling to the top and bottom of the page.
- Blocking Firefox's built-in Alt+F to open the File menu.
- Blocking Firefox's built-in Alt+V to open the View menu.
- Blocking Firefox's built-in Alt+S to open the History menu.
- Blocking Firefox's built-in Alt+H to open the Help menu.

I have already disabled both search and command mode shortcuts in the GitHub accessibility settings. It doesn't help. Your keyboard interception persists, breaking basic browser functionality and making almost every visit to a GitHub repo a frustrating experience.

The present behavior is the opposite of accessibility. It makes me want to cry.

Please, for the love  of sanity, will you give us a way to disable this madness completely?

**Comment by ghostinhershell** [comment]
Hi @foresto, 

Thanks so much for reaching out in our community! You can disable character key shortcuts, while still allowing shortcuts that use modifier keys, in your accessibility settings. For more information, see "[Managing accessibility settings](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/managing-accessibility-settings)."

**Reply by foresto** [reply]
I have already done that, as stated in my post. It doesn't fix the problem, because more browser-breaking shortcuts remain that cannot be disabled. That's the problem.

**Comment by sabo-t** [comment]
Let us disable all keyboard shortcuts please. People use extensions for enabling a keyboard-centric workflow (vimium-c, tridactyl), and it is very annoying when your mappings take precedence over the mappings defined in such extensions. Please lower the precedence if possible or allow us to disable all shortcuts.

**Comment by tyleralbee** [comment]
It isn't a good experience using GitHub's GUI when using Vimium. On the Chrome webstore, Vimium has capped out at 500.000 users and I would imagine a significant portion of them use GitHub too. 

Does anyone have a workaround?
